examples/config/dialog_on_exit_shutdown.config

```diff
@@ -0,0 +1,27 @@
+<?xml version="1.0"?>
+
+<!-- Window -->
+<window>
+  <width>1000</width>
+  <height>845</height>
+  <default_exit_action>SHUTDOWN_SERVER</default_exit_action>
+  <dialog_on_exit>true</dialog_on_exit>
+  <dialog_on_exit_options>
+    <prompt_text>really? žřčš</prompt_text>
```

Comment:
I've put it there to make sure UTF-8 is interpreted correctly. But it can be removed now that we know it is... or it could stay so that everyone who runs the example will see that the config is UTF-safe.

Reply:
removed the diacritics characters